cmd/flags.go

```diff
@@ -349,6 +349,10 @@ func metaCacheFlags(defaultEntryCache float64) []cli.Flag {
 			Value: "1.0s",
 			Usage: "dir entry cache timeout",
 		},
+		&cli.BoolFlag{
+			Name:  "readdir-cache",
+			Usage: "enable kernel caching of readdir entries, with timeout controlled by attr-cache flag",
```

Comment:
(require linux kernel 4.20+)

Reply:
attr-cache should be enabled, otherwise the readdir will be slow than before, because there will be lots of lookup/getattr operations.

Reply:
How to ensure `attr-cache` is enabled? set it to 1s or use `logger.Fatalf` to force user specify one in `Config.SelfCheck()`?

Reply:
We can mention this in docs

Reply:
Updated code to print a warning log if the readdir-cache dependency is not met.